hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/UploadContentProviders.java

```diff
@@ -275,7 +283,10 @@ public final InputStream newStream() {
       checkOpen();
       streamCreationCount++;
       if (streamCreationCount > 1) {
-        LOG.info("Stream created more than once: {}", this);
+        LOG.info("Stream recreated: {}", this);
+        if (LOG.isDebugEnabled()) {
+          LOG.debug("Stream creation stack", new Exception("here"));
```

Comment:
I think this is for some testing,

Reply:
good q. should we delete it? it's actually pretty handy for troubleshooting networking issues